docs/UserGuide.md

```diff
@@ -3,10 +3,13 @@ layout: page
 title: User Guide
 ---
 
-Productiv is a **desktop app for product owners/managers to oversee the development 
-of their product, optimized for use via a Command Line Interface** (CLI) while still 
-having the benefits of a Graphical User Interface (GUI). If you can type fast, Productiv 
-can get your product management tasks done faster than traditional GUI apps.
+*Productiv* is a desktop app that assists product managers like yourself with organising product-related information 
+(**contacts**, **deliverables** and **meetings**) so that you can track your product's development more comprehensively 
+and conveniently.
+
+*Productiv* is optimized for use via Command Line Interface (CLI), but still has the benefits of a Graphical User 
+Interface (GUI). Thus, if you can type fast, *Productiv* can get your product management tasks done faster than 
+traditional GUI apps.
 
 * Table of Contents
 {:toc}
```

Comment:
Consider removing this section if there's no intention of updating Table of Contents for now.